### ISSUE TITLE
Read the Kaggle Knowhow articles

### ISSUE THREAD

**Issue by jiwoncpark** [jiwoncpark/deep-learning-notebooks#6]
Kaggle Knowhow 2-part articles (https://github.com/zzsza/Kaggle-knowhow/blob/master/01.Kaggle-Intro.md and https://github.com/zzsza/Kaggle-knowhow/blob/master/02.Kaggle-Flow.md). 

- Usually submit two entries, record hyperparameters, etc. in the description field of submission, and .zip files are good for download and upload speed.
- Look at the evaluation metric (e.g. for accuracy, MAP/MSE/log error).
- Do an exploratory data analysis (EDA). Reference kernels but also your own, e.g. groupby each column and do a feature count, plot distributions, find missing data and think about substitution methods.
- Data preprocessing for missing data, outliers, and nominal variables. Adjust each data size or data augmentation.
- Implement a single model, e.g. random forest, xgboost, LGBM, Catboost
- Feature engineering is important!!! Apply many transformations and see if model improves. Iterate many times.
- Try many architectures and also try stacking predictions from various models, e.g. 0.3*model_A_pred + 0.7*model_B_pred. See StackNet.
